pkg/model/model.go

```diff
@@ -280,10 +281,15 @@ func (p RepeaterUntilProcess) ProcessDictionary(dictionary Dictionary, out Colle
 	out.Collect(dictionary)
 
 	var output bytes.Buffer
-
-	err := p.tmpl.Execute(&output, dictionary.Unordered())
+	var err error
+	defer func() {
+		if r := recover(); r != nil {
+			err = fmt.Errorf("Cannot execute template, error: %v", r)
+		}
+	}()
+	err = p.tmpl.Execute(&output, dictionary.Unordered())
 	if err != nil {
-		return err
+		log.Err(err).Msg("Error while executing template")
```

Comment:
Is it ok to not return error here ?

Reply:
If an error occurs, the repeat process will stop. It's needed when template.Execute returns error of comparison type (`{{eq .name "Joseph"}}` will fail if `.name` is type null.

We can keep the function to return an error, except when `--skip-line-on-error` is active ( or --skip-field-on-error)